src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -98,7 +98,7 @@ internal class CultureNameResourceSetPair
             public ResourceSet? lastResourceSet;
         }
 
-        protected string? BaseNameField;
+        protected string BaseNameField = string.Empty;
```

Comment:
Why is the initialization needed? As far as I can tell, the field is initialized to non-null by all constructors.

Reply:
Ok, I see the default constructor does not initialize it. I think it would be better to initialize it just in the default constructor.

Reply:
But the default constructor is protected. Someone that would override it, wouldn’t be forced to initialize it?

Reply:
You have to call base class constructor from your constructor. If you do not call it explicitly, the C# compiler will call it for you as the first thing in your constructor.

Reply:
Ok. Cool. Then I’ll initialize it there. Didn’t know that.

Reply:
Fixed.